ansible/inventories/production/group_vars/all/vars.yml

```diff
@@ -20,7 +20,6 @@ catalog_ckan_service_url: https://{{ catalog_host_public }}
 catalog_ckan_admin_service_url: https://{{ catalog_host_admin }}
 catalog_ckan_solr_port: "8983"
 catalog_ckan_solr_host: datagov-solr1p.prod-ocsit.bsp.gsa.gov
-catalog_ckan_wsgi_processes: 6
```

Comment:
Are you sure you want to change this? On catalog-classic, I think we're running m4.2xlarge with 8 vcpus and 32G of ram. That would default to 17 ckan processes which tend to be memory hungry. Are you sure that catalog-classic can handle that?

Reply:
I'm not sure which is the right number of workers for each context. Maybe a load test could help to define it.
For production, I leave `6` as it was before.
Considering this, I added a `4` for stagging and `2` for sandbox.